frontend/src/pages/history-page/history-page.tsx

```diff
@@ -24,28 +24,29 @@ const HistoryPage: FC = () => {
     return state.theme.isLightTheme;
   });
 
+  const historyData = useAppSelector((state) => {
+    return state.history.data;
+  });
+
+  const { isFirstHistoryPageLoad, currentPage, lastPage, dataStatus } = historyData;
```

Comment:
you can apply destructuring on the line where you get historyData

Reply:
done